src/borg/locking.py

```diff
@@ -21,6 +23,29 @@ def get_id():
     return _hostname, _pid, thread_id
 
 
+def check_lock_data(host, pid, thread):
+    """Check if the host, pid, thread combination corresponds to a dead process on our local node or not."""
+    if host != _hostname:
+        return False
+
+    try:
+        # This may not work in Windows.
```

Comment:
I have read that at first kill(pid, 0) used to kill a process in windows for good. And then that was changed and it started to return error at all times.
So this nees to be verified.